config/core/test-configs.yaml

```diff
@@ -2411,6 +2436,10 @@ test_configs:
     test_plans:
       - baseline_qemu
 
+  - device_type: qemu_x86_64-uefi-chromiumos
+    test_plans:
+      - chromiumosvm-tast
```

Comment:
Might be worth adding baseline_qemu, as a reference point if Chrome OS jobs fail to boot for some reason.

Reply:
Will do